src/Admin/ReaderThemes.php

```diff
@@ -210,17 +227,38 @@ public function get_default_reader_themes() {
 				]
 			);
 
-			if ( ! is_wp_error( $response ) ) {
-				set_transient( $cache_key, $response, DAY_IN_SECONDS );
+			if ( is_array( $response ) ) {
+				$response = (object) $response;
 			}
```

Comment:
Is this because WP can return both an array and an object and you want to avoid catering for both cases?

Reply:
Yes, I think `themes_api` is likely to be an object here, but that function makes an external request and is dealing with an API version. It's not entirely transparent to me. But if we think `themes_api`, without filters, will always reliably return a stdClass object or a WP_Error, then we could remove this.

Reply:
There's an old WordPress saying: "Anything that can be filtered will be filtered to be unusable" - better safe than sorry.